conn/pool.go

```diff
@@ -119,6 +125,7 @@ func (p *Pools) getPool(addr string) (*Pool, bool) {
 	return existingPool, has
 }
 
+// Connect creates a Pools instance for the node with the given address or returns the existing one.
```

Comment:
I think this creates a `Pool` instance, not a `Pools`

Reply:
Done.